tests/inputFields.spec.ts

```diff
@@ -5,91 +5,84 @@ test.beforeEach( async({page}) => {
-  test.slow()
-  const rabbit = page.locator('input[name=pettype_name]').first().inputValue()
-  expect(await rabbit).toBe('rabbit')
-  
+  expect(await page.locator('input[name=pettype_name]').first().inputValue()).toEqual('rabbit')
   
   //7. Click on "Edit" button for the same "rabbit" pet type
   await page.getByText('Edit').first().click()
   
   //8. Change the pet type name back from "rabbit" to "cat" and click "Update" button
-  await textField.click()
-  await textField.clear()
-  await textField.pressSequentially('cat')
+  await editPetTypeField.click()
+  await editPetTypeField.clear()
+  await editPetTypeField.fill('cat')
   await page.getByText('Update').click()
   
   //9. Add the assertion that the first pet type in the list of names has a value "cat" 
-  const catEntry = page.locator('input[name=pettype_name]').first().inputValue()
-  expect(await catEntry).toBe('cat')
+  const updatedPetType = page.locator('input[name=pettype_name]').first().inputValue()
+  expect(await updatedPetType).toBe('cat')
 })
 
 test('Cancel pet type update', async ({page}) => {
   //3. Click on "Edit" button for the "dog" pet type
   //assumes "dog" is always second
   await page.getByText('Edit').nth(1).click()
-  const textField = page.locator('#name')
+  const editPetTypeField = page.locator('#name')
 
   //4. Type the new pet type name "moose"
-  await textField.click()
-  await textField.clear()
-  await textField.pressSequentially('moose')
+  await editPetTypeField.click()
+  await editPetTypeField.clear()
+  await editPetTypeField.fill('moose')
 
   //5. Add assertion the value "moose" is displayed in the input field of the "Edit Pet Type" page
-  expect(await textField.inputValue()).toBe('moose')
+  expect(await editPetTypeField.inputValue()).toBe('moose')
   
   //6. Click on "Cancel" button
   await page.getByText('Cancel').click()
 
   //7. Add the assertion the value "dog" is still displayed in the list of pet types
-  const dogEntry = page.locator('input[name=pettype_name]').nth(1).inputValue()
-  expect(await dogEntry).toBe('dog')
+  expect(await page.locator('input[name=pettype_name]').nth(1).inputValue()).toBe('dog')
 })
 
 test('Pet type name is required validation', async ({page}) => {
-
-  test.slow()
   //3. Click on "Edit" button for the "lizard" pet type
+  //assumes lizard is always third
   await page.getByText('Edit').nth(2).click()
-  const textField = page.locator('#name')
+  const editPetTypeField = page.locator('#name')
 
   //4. On the Edit Pet Type page, clear the input field
-  await textField.click()
-  await textField.clear()
+  await editPetTypeField.click()
+  await editPetTypeField.clear()
 
   //5. Add the assertion for the "Name is required" message below the input field
-  expect(await page.getByText('Name is required').isVisible()).toBeTruthy()
+  expect(await page.locator('div').filter({ hasText: 'Name' }).nth(-1).textContent()).toEqual('Name is required')
```

Comment:
It's better. The only dependency on the index is not good here.
With a locator assertion and a more optimized locator, it can look like this:
`await expect(page.locator('div', {hasText: 'Name'}).locator('.help-block')).toHaveText('Name is required')`